### ISSUE TITLE
Filewarnings without baseline results in zero warnings instead of skipping

### ISSUE THREAD

**Issue by gerwinjansen** [MicrosoftPremier/VstsExtensions#100]
... so the build fails because the current build has more warnings than the previous (erroneous) zero.

### Steps to reproduce
1. Have a master branch with a C# project with 1 compiler warning, and a file with 1 warning.
1. Configure BQC as this (change the regex according to your own file):

    ```yaml
    - task: BuildQualityChecks@7
    inputs:
        checkWarnings: true
        showStatistics: true
        evaluateFileWarnings: true
        warningFiles: 'modelchecker.csv'
        warningFileFilters: '/(?<object>^\w+,[\w\.]*,[\w\.]+?),(?<category>\w+,\w+?),(?<message>[^,]+ ?)/'
        warningFilesArtifact: 'modelcheckerwarnings'
        includePartiallySucceeded: false
    ```
1. Run a build on master, use BQC.ForceNewBaseline=true as needed
1. Create a new branch from master
1. Run a build on the new branch

### Actual result
The build fails: `[ERROR] The number of warnings (2) has increased since the previous build! The previous build had 0 warning(s).`

### Expected result
`[WARNING] The baseline build could not be found. All policies based on the previous build will pass.`

### Debug log
with System.Debug=true and BQC.LogRawData=true: [Rawlog.txt](https://github.com/MicrosoftPremier/VstsExtensions/files/4817878/Rawlog.txt)

**Comment by ReneSchumacher** [comment]
Hi Gerwin,

thanks for reporting this. This is indeed a bug in the task. We'll fix that with the next version (most probably this week).

Thanks,
René

**Comment by ReneSchumacher** [comment]
Hi again,

the issue is fixed with v7.2.1 which is going live right now.

Happy building and keep up the very good feedback! Thanks a lot for that!

René
